annotations.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	mkr "github.com/mackerelio/mackerel-client-go"
+	"github.com/mackerelio/mkr/logger"
+	"gopkg.in/urfave/cli.v1"
+	"os"
+)
+
+var commandAnnotations = cli.Command{
+	Name: "annotations",
+	Subcommands: []cli.Command{
+		{
+			Name:        "create",
+			Usage:       "create annotation",
+			Description: "Creates an annotation.",
+			Action:      doAnnotationsCreate,
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "title", Value: "", Usage: "Title for annotation."},
```

Comment:
I think period is not needed for flag description especially if it's not a sentence.